### PR TITLE
remove index column from -raw output

### PR DESCRIPTION
**Pull request by atravitz**
As discussed as a team earlier, I'm removing the `repeat` column from the `gather --raw` output. We may add a `--debug` mode that adds both repeat indexes and json filepaths in a future PR.

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

**Review by IAlibay — APPROVED** [review]
lgtm thanks!

Could you add in a rever entry for the changelog please?

**Comment by atravitz** [comment]
@IAlibay done. this is my first time using rever for changelogs so let me know if it doesn't look correct.

File: news/fix_missing_replicates.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* fixed bug where `openfe gather --report raw` was only including first replicates.
```
Comment (mikemhenry) [inline news/fix_missing_replicates.rst:19]: LGTM!

Comment (IAlibay) [inline news/fix_missing_replicates.rst:19]:
```suggestion
* Fixed bug where `openfe gather --report raw` was only including first replicates.
```
Sorry being pedantic - capitalization on a line with a full stop :)

**Pull request merged**